cookbook/ui_customization/create_a_reference_data_crud.rst

```diff
@@ -0,0 +1,176 @@
+How to create the screens to manage a Reference Data
+====================================================
+
+.. note::
+    The code inside this cookbook entry requires you to install the :doc:`/reference/bundles/custom_entity_bundle/index`.
+
+Creating the Reference Data Management Screens
+**********************************************
+
+The Grid
+--------
+
+To benefit from the grid component (which comes natively with filtering and sorting),
+you can define the vendor grid as following:
+
+.. code-block:: yaml
+
+    # /src/Acme/Bundle/AppBundle/Resources/config/datagrid.yml
+    datagrid:
+        color:
+            options:
+                entityHint: color
+            source:
+                type: pim_datasource_default
+                entity: Acme\Bundle\AppBundle\Entity\Color
+                repository_method: createDatagridQueryBuilder
+            columns:
+                code:
+                    label: Code
+                name:
+                    label: name
+            properties:
+                id: ~
+                show_link:
+                    type: url
+                    route: pim_customentity_show
+                    params:
+                        - id
+                        - customEntityName
+                edit_link:
+                    type: url
+                    route: pim_customentity_edit
+                    params:
+                        - id
+                        - customEntityName
+                delete_link:
+                    type: url
+                    route: pim_customentity_delete
+                    params:
+                        - id
+                        - customEntityName
+            actions:
+                show:
+                    type:      navigate
+                    label:     Show the reference data
+                    icon:      eye-open
+                    link:      show_link
+                edit:
+                    type:      navigate
+                    label:     Edit the reference data
+                    icon:      edit
+                    link:      edit_link
+                    rowAction: true
+                delete:
+                    type:  delete
+                    label: Delete the reference data
+                    icon:  trash
+                    link:  delete_link
+            filters:
+                columns:
+                    code:
+                        type:      string
+                        label:     Code
+                        data_name: rd.code
+                    name:
+                        type:      string
+                        label:     Name
+                        data_name: rd.name
+            sorters:
+                columns:
+                    code:
+                        data_name: rd.code
+                    name:
+                        data_name: rd.name
+                default:
+                    code: '%oro_datagrid.extension.orm_sorter.class%::DIRECTION_ASC'
```

Comment:
What is missing here is a reference documentation about grid link, see https://github.com/akeneo/pim-docs/issues/490